tests/e2e/cucumber/steps/ui/adminSettings.ts

```diff
@@ -481,13 +486,15 @@ When(
   ): Promise<void> {
     const { page } = this.actorsEnvironment.getActor({ key: stepUser })
     const groupsObject = new objects.applicationAdminSettings.Groups({ page })
+    const groups = []
 
     switch (actionType) {
       case 'batch actions':
         for (const { group } of stepTable.hashes()) {
+          groups.push(group)
           await groupsObject.selectGroup({ key: group })
```

Comment:
maybe collect ids in the methods select users or select groups? 
`await groupsObject.selectGroup({ key: group })` and return ids array
then you don't need collect groups/users by name in the adminSettings and convert it to idarray in other place

Reply:
Can do that. Thanks :+1: